src/main/resources/com/google/api/codegen/php/test.snip

```diff
@@ -102,27 +127,39 @@
         @end
     @end
     
-    @switch test.clientMethodType
-    @case "OptionalArrayMethod"
-        {@optionalArraySuccessAsserts(test)}
-    @case "PagedOptionalArrayMethod"
-        {@pagedOptionalArraySuccessAsserts(test)}
-    @default
-        $unhandled case: {@test.clientMethodType.toString}$
-    @end
 @end
 
-@private optionalArraySuccessAsserts(test)
+@private successAsserts(test)
+    @if test.hasReturnValue
+        $response = $client->{@test.clientMethodName}({@sampleMethodCallArgList(test.initCode.fieldSettings)});
+        
+    @else
+        $client->{@test.clientMethodName}({@sampleMethodCallArgList(test.initCode.fieldSettings)});
+        
+    @end
+    {@singleCallSuccessAsserts(test)}
     @if test.hasReturnValue
         $this->assertEquals($expectedResponse, $response);
     @end
 @end
 
-@private pagedOptionalArraySuccessAsserts(test)
+@private pagedSuccessAsserts(test)
```

Comment:
This name is confusing - should it be something like `pagedMethodSuccessAsserts`?